src/pages/billing/Project.js

```diff
@@ -48,17 +48,17 @@ const getBillingAccountIcon = status => {
 }
 
 const WorkspaceCardHeaders = memoWithName('WorkspaceCardHeaders', ({ needsStatusColumn, sort, onSort }) => {
-  return div({ style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem', marginBottom: '0.5rem' } }, [
+  return div({ role: 'row', style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem', marginBottom: '0.5rem' } }, [
```

Comment:
There were a number of issues causing failures. Basically the workspaces data was being rendered as a list, but a table is a more accurate representation, as there are sortable column headers. The view is still funky due to the expand icon at the end of the row that shows additional details (not in any of the columns), but with these changes at least the sorting indicators read properly and the number of rows (workspaces) reads correctly.

These changes make this view consistent with the user tab, which uses `MemberCardHeader` and `MemberCard` with these aria roles defined.